hospital_management_system/hospital_app/serializers.py

```diff
@@ -26,35 +27,18 @@ class Meta:
                   'address')
 
 class PatientSerializer(serializers.ModelSerializer):
-    #id = serializers.UUIDField()
-    #hospital_details = serializers.PrimaryKeyRelatedField(source="hospital",many=False,queryset=Hospital.objects.all())
-    #hospital_name = HospitalSerializer(source='hospital',read_only=True)
-
+    
     class Meta:
         model = Patient
-        fields =(
-                 'patient_name',
-                 'contact_number',
-                 'patient_address',
-                 'department',
-                 'disease')    
+        
+        fields = '__all__'    
```

Comment:
since no data changes are done after fetching from serializer,
instead of serializer can use "objects.all()"

Reply:
Fixed